clients/imodels-client-management/src/operations/briefcase/BriefcaseOperations.ts

```diff
@@ -69,18 +71,20 @@ export class BriefcaseOperations<TOptions extends OperationOptions> extends Oper
   public async getSingle(params: GetSingleBriefcaseParams): Promise<Briefcase> {
```

Comment:
Would be nice to pass `x-correlation-id` header in all tests that call client operations. I think all client operations are called at least once, adding `x-correlation-id` header would simplify troubleshooting test failures and would ensure that one can pass headers in all client methods.